frontend/src/components/ConfirmDialog/styled.ts

```diff
@@ -1,11 +1,22 @@
 import styled from 'styled-components';
 import { Dialog } from '@mui/material';
 
-export const StyledDialog = styled(Dialog)({
+interface IDialog {
+  width?: number | undefined;
+  height?: number | undefined;
+}
+
+export const StyledDialog = styled(Dialog)<IDialog>(({ width, height }) => ({
   '& .MuiDialog-paper': {
     borderRadius: '5px',
     boxShadow: 'none',
     minWidth: '660px !important',
     maxHeight: '335px !important',
+    ...(width && {
```

Comment:
You can create variants of custom component, it will help you to pass one prop `variant` instead of `width` and `height` in your own component and will help to avoid unnecessary constant `CONFIRM_SIZE` in HeaderContainer

Reply:
Its is not problem, good idea! I Will use `export const SIZE = {
  small: 'small',
  medium: 'medium',
  large: 'large',
  xlarge: 'xlarge',
};
` from our constants